version.sbt

```diff
@@ -1 +1,3 @@
-version in ThisBuild := "4.0.0-SNAPSHOT"
+enablePlugins(GitVersioning)
```

Comment:
sbt eclipse also, for some reason, has TWO projects.  YOu probably need to do this in both projects.